src/go-build-wrapper

```diff
@@ -27,7 +27,22 @@ if ! cd "$1"; then
     exit 1
 fi
 
-go build -trimpath -ldflags "-extldflags '-Wl,-rpath,/run/host/usr/lib -Wl,-rpath,/run/host/usr/lib64' -linkmode external -X github.com/containers/toolbox/pkg/version.currentVersion=$3" -o "$2/toolbox"
+if ! libc_dir=$("$4" --print-file-name=libc.so); then
```

Comment:
I wish there was a way to do this with Meson's [compiler.find_library](https://mesonbuild.com/Reference-manual_returned_compiler.html#compilerfind_library).